database.cc

```diff
@@ -1534,6 +1534,15 @@ future<> database::apply(schema_ptr s, const frozen_mutation& m, db::commitlog::
     return update_write_metrics(_apply_stage(this, std::move(s), seastar::cref(m), timeout, sync));
 }
 
+future<> database::apply_hint(schema_ptr s, const frozen_mutation& m, db::timeout_clock::time_point timeout) {
+    if (dblog.is_enabled(logging::log_level::trace)) {
+        dblog.trace("apply hint {}", m.pretty_printer(s));
+    }
+    return with_scheduling_group(_dbcfg.streaming_scheduling_group, [this, s = std::move(s), &m, timeout] () {
```

Comment:
If you don't declare a lambda `mutable` std::move(X) on X from the capture list is going to be a copy because X is going to be const. In your case X is an `s`  (schema_ptr).

Reply:
Oh, now I see why it was needed in other places. Thanks, fixed.